### DISCUSSION TITLE
Any plans to fix the build and test bot?

### DISCUSSION THREAD

**Discussion by RusseII** [chand1012/Discord-Quick-Meme#22] [Q&A]
![image](https://user-images.githubusercontent.com/15036618/102560440-40245500-40a0-11eb-9c2d-d5f104f89110.png)

**Comment by chand1012** [comment]
It was actually an issue with an external API unit test:

![image](https://user-images.githubusercontent.com/3521582/102572679-b9c93c80-40ba-11eb-8997-3ce88115f7af.png)

In the develop branch, it works fine. I fixed it somewhere along the line, but I can't find which commit did it. Hopefully when I merge to master in the coming weeks it will get resolved.
